src/main/java/team4/footwithme/chat/api/request/ChatroomRequest.java

```diff
@@ -0,0 +1,13 @@
+package team4.footwithme.chat.api.request;
+
+import jakarta.validation.constraints.NotNull;
+import team4.footwithme.chat.service.request.ChatroomServiceRequest;
+
+public record ChatroomRequest(
+        @NotNull(message = "채팅방 이름은 필수입니다.")
+        String name
```

Comment:
```suggestion
        @NotNull(message = "채팅방 이름은 필수입니다.")
        String name
```

여기두 한번 생각해보시면 좋을 것 같아요 ㅎㅎ